File: fcB2002/fcB2002.c
int fcB2002(int a, int b, int c)
{
	int result = 0;
	if (c > b)
	{
		if (b > a)
		{
			result = 1;
			int t = a + b;
			if (t < c)
			{
				result = 2;
				int t2 = a * b;
				if (((t2 - c) <= 5) && ((t2 - c) >= 0))
				{
					result = 3;
				}
			}
			else
			{
				result = 4;

			}
		}
	}
	return result;
}

File: fcB2002/fcB2002.php
<?php
$f1 = $_post['f1value'];
$f2 = $_post['f2value'];
$f3 = $_post['f3value'];
$result = 0;

if($f3 > $f2)
{
	if($f2 > $f1)
	{
		$result = 1;
		$t = $f1 + $f2;
		if($t < $f3)
		{
			$result = 2;
			$t2 = $f1 * $f2;
			if ((($t2 - $f3) <= 5) && (($t2 - $f3) >= 0))
			{
				$result = 3;
			}
		}
		else
		{
			$result = 4;
		}
	}
}

?>
